src/decorators/debounce.ts

```diff
@@ -1,6 +1,12 @@
-import lodashDebounce, {DebounceSettings} from 'lodash/debounce';
+import debounce from 'lodash/debounce';
 
-export default function debounce(wait = 500, options?: DebounceSettings) {
+export interface DebounceSettings {
+    leading?: boolean,
+    maxWait?: number,
+    trailing?: boolean,
+}
+
+export default function debounceDecorator(wait = 500, options?: DebounceSettings) {
```

Comment:
`src/decorators/debounce.ts[1, 8]: Expected import 'lodashDebounce' to match the default export 'debounce'.`